html_text/html_text.py

```diff
@@ -135,27 +142,29 @@ def selector_to_text(sel, guess_punct_space=True, guess_layout=True):
     See html_text.extract_text docstring for description of the approach
     and options.
     """
+    import parsel
```

Comment:
I'd suggest to add a "Convert a cleaned parsel.Selector to text." to the docstring, to make it clear that this part works with parsel (and needs it).

Reply:
done, a good idea!